src/Akka.Logger.Serilog/SerilogMessageFormatter.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Akka.Event;
+using Serilog.Events;
+using Serilog.Parsing;
+
+namespace Akka.Logger.Serilog
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// This class contains methods used to convert Serilog templated messages
+    /// into normal text messages.
+    /// </summary>
+    public class SerilogLogMessageFormatter : ILogMessageFormatter
```

Comment:
Brought the `SerilogLogMessageFormatter` back - necessary for compatibility with the non-Serilog stuff that might touch any of the `LogEvent` instances.